image/image.css

```diff
@@ -0,0 +1,31 @@
+:root {
+    --primary-color: white;
+    --danger-color:red;
+    --warning-color: orange;
+    --success-color: rgb(92, 218, 20);
+  }
+body {
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    background-color: rgb(187, 181, 175);
+}
+
+.html {
+font-size: 62.5%;
+}
+
+.img-container {
+    margin:5px;
+}
+
+.img {
+    width: 20%;
+    height: 20%;
```

Comment:
Read about object-fit and object-position to style the images in a efficient way.